curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f1a00ac619ddc1e259a66.md

```diff
@@ -32,7 +32,7 @@ assert.match(__helpers.removeJSComments(code), /i\s*<\s*count/);
 Your `for` loop should use `i < count` as the condition.
 
 ```js
-assert.match(__helpers.removeJSComments(code), /for\s*\(\s*let\s+i\s*=\s*0;\s*i\s*<\s*count;/);
+assert.match(__helpers.removeJSComments(code), /for\s*\(\s*let\s+i\s*=\s*0;\s*i\s*<\s*count\s*;/);
```

Comment:
```suggestion
assert.match(__helpers.removeJSComments(code), /for\s*\(\s*let\s+i\s*=\s*0\s*;\s*i\s*<\s*count\s*;/);
```
In case it gets somehow changed.